openfecwebapp/templates/partials/filings-filter.html

```diff
@@ -20,7 +20,7 @@
 
   <button type="button" class="js-accordion-trigger accordion__button">Filing date</button>
   <div class="accordion__content">
-    {{ years.years('cycle', 'Two-year period')  }}
+    {{ years.years('cycle', 'Years active')  }}
```

Comment:
To me, "Years Active" doesn't make sense for filings since it is about particular records and not the activity of a group. In this case, cycle is derived from the year the filing applied to. Can we say "Report year" or something like that?